nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -238,3 +228,34 @@ def _create_quantizer_insertion_command(nncf_graph: NNCFGraph,
         quantizer = PTMinMaxAlgoBackend._create_quantizer(quantizer_config,
                                                           scale_shape, statistics)
         return PTInsertionCommand(target_point, quantizer, TransformationPriority.QUANTIZATION_PRIORITY)
+
+    @staticmethod
+    def get_model_type_ignore_scope(model_type: ModelType):
+        if model_type == ModelType.TRANSFORMER:
+            types = []
+            metatypes_to_add = [om.PTAddMetatype, om.PTPowerMetatype, om.PTPowerMetatype,
+                                om.PTMulMetatype, om.PTSubMetatype, om.PTMeanMetatype]
+            type_name_to_add = ["squeeze"]
+            for metatype in metatypes_to_add:
+                for f_names in metatype.module_to_function_names.values():
+                    types.extend(f_names)
+            types.extend(type_name_to_add)
+            return IgnoredScope(types=types)
+        return IgnoredScope()
+
+    @staticmethod
+    def get_weight_nodes(nncf_graph: NNCFGraph) -> List[NNCFNode]:
+        return nncf_graph.get_nodes_by_metatypes([
+            om.PTModuleConv1dMetatype,
+            om.PTModuleConv2dMetatype,
+            om.PTModuleConv3dMetatype,
+            om.PTDepthwiseConv1dSubtype,
+            om.PTDepthwiseConv2dSubtype,
+            om.PTDepthwiseConv3dSubtype,
+            om.PTModuleLinearMetatype,
+            om.PTModuleConvTranspose1dMetatype,
+            om.PTModuleConvTranspose2dMetatype,
+            om.PTModuleConvTranspose3dMetatype,
+            om.PTModuleEmbeddingMetatype,
+            om.PTModuleEmbeddingBagMetatype,
```

Comment:
Don't we need to check `layer_attributes` also?

Reply:
Done